stacks/knot-cloud/README.md

```diff
@@ -0,0 +1,165 @@
+# knot-cloud
+
+### Configure DNS
+
+This stack uses [Traefik](https://traefik.io) as a reverse proxy for the services and requires that you configure your DNS server to point, at least, the **www**, **ws**, **auth** and **bootstrap** subdomains to the machine where the stack is being deployed, so that it can route the requests to the appropriated service. It is possible to configure it to route by path or port, but instructions for that won't be provided here for brevity.
+
+If you don't have a domain or can't configure the main DNS server, you can configure a test domain in your machine before proceeding. Either setup a local DNS server, e.g. [bind9](https://wiki.debian.org/Bind9), or alternatively update your hosts file to include the following addresses:
+
+```
+127.0.0.1	www
+127.0.0.1	ws
+127.0.0.1	auth
+127.0.0.1	bootstrap
+```
+
+On Windows, the hosts file is usually located under `c:\Windows\System32\Drivers\etc\hosts`. On Unix systems, it is commonly found at `/etc/hosts`. Regardless of you operating system, administrator or super user privileges will be required.
+
+Notice that when deploying KNoT Cloud locally, most of the times the `<your-domain>` subdomains in the following sections should be disregarded. For instance, you would access your KNoT Cloud at `https://www` after deploy.
+
+### Deploy: stage 1
+
+Stage 1 contains the core services. The next sections provide the instructions to configure and deploy them. Whenever a configuration file is mentioned, it refers to a file found at `stacks/knot-cloud/prod/env.d`, for production mode, or `<path>/stack/dev/env.d`, for development mode.
```

Comment:
The path should be updated & we don't have neither `prod` for `knot-cloud-core` nor a `dev` being generated in `knot-cloud init knot-cloud` :grimacing: 

```suggestion
Stage 1 contains the core services. The next sections provide the instructions to configure and deploy them. Whenever a configuration file is mentioned, it refers to a file found at `stacks/knot-cloud/prod/env.d`, for production mode, or `<path>/stack/env.d`, for development mode.
```

Similarly to the previous discussion, why is there a `/dev` dir with env. files if the development one isn't `dev`, but actually `<path>/stack/env.d`? I dont think theres a reason to generate config files other then when specifying a different directory (outside knot-cloud repo root).

Reply:
Same answers.